Draft: Guidelines for speaking on behalf of the project (#696)
graphql/graphql-wg: GraphQL-TSC.md

### Speaking on behalf of the GraphQL Project

> GraphQL Contributors as individuals are welcome to express opinions and messages of political advocacy in any of their personal spaces (e.g. personal Twitter, blog) as well as within GitHub issues and PR discussions, provided it adheres to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md). When speaking on behalf of the GraphQL project (e.g. technical docs, project websites, project Twitter, READMEs, or anything merged into a git repository under the GraphQL organization) we only allow posting opinions or messages of political advocacy using the voice of the project with prior approval via GraphQL TSC vote.

Reviewer: The final sentence is a little repetative ("speaking on behalf [...] voice of the project") and the parenthesis in the middle makes it a little disjoint. I've had a go at editing it to address these concerns:

```suggestion
GraphQL Contributors as individuals are welcome to express opinions and messages of political advocacy in any of their personal spaces (e.g. personal Twitter, blog) as well as within GitHub issues and PR discussions, provided it adheres to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md). We require approval via GraphQL TSC vote prior to posting opinions or messages of political advocacy using the voice of the GraphQL project (e.g. within technical docs, project websites, project Twitter, READMEs, or anything merged into a git repository under the GraphQL organization).
```

> In addition to prior approval from the TSC, all expressed opinions or messages of political advocacy from the GraphQL project should be clear about their purpose and intent, use the appropriate surfaces and channels, be paired with tangible action from the project, and provide a clear call-to-action for community members.

Reviewer: ```suggestion
In addition to prior approval from the TSC, all expressed opinions or messages of political advocacy from the GraphQL project should be clear about their purpose and intent, use the appropriate surfaces and channels, be paired with tangible action from the project, provide a clear call-to-action for community members, and specify the timeframe during which they will be removed\*.

\* _Complete removal may never be possible due to the nature of `git`, `npm`, and other channels; as such, we're referring to removal from the primary communication channels ─ the latest versions of the website, READMEs, etc._
```

Author: Added inline

> GraphQL Contributors as individuals are welcome to express opinions and messages of political advocacy in any of their personal spaces (e.g. Twitter, blog) as well as within GitHub issues and PR discussions, provided it adheres to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md). However when speaking on behalf of the GraphQL project (e.g. technical docs, project websites, READMEs, or anything merged into a git repository under the GraphQL organization) we do not allow posting opinions or messages of political advocacy using the voice of the project without prior approval via GraphQL TSC vote.

Reviewer 2: I’m still stuck on second sentence. If the scope of the code of conduct includes outside of GitHub, when speaking as voice of project only, I think you need an “In addition,” and not a “however.” I would also add an example in the list from outside GitHub. And, minor, I would also change “we do not allow” to “we allow with permission,” just style, keeping it positive :)

Author: Good suggestions. I just dropped the "However", it's not bringing any clarity. I rephrased the latter to avoid a negative phrasing.

> In addition to prior approval from the TSC, any expressed opinion or message of political advocacy from the GraphQL project should meet these guidelines around message, delivery, and follow-through. All messages should be clear and coordinated across available surfaces and channels. Advocacy messages should be paired with tangible action from the project as well as a clear call-to-action for community members.

Reviewer 3: This is a nit as I tripped up on the meaning on my first read through.

> should meet these guidelines around message, delivery, and follow-through.

I think this preamble is redundant.

Possibly easier to ensure messages achieve all of the requirements if they're presented as a "check list" of bullet points.
```
All political advocacy or opinion messages should be:
- clear
- coordinated across available surfaces and channels
- paired with tangible action from the project
- providing a clear call-to-action for community members
```

Author: This is great feedback. I'll incorporate. This is minor, but I'm hoping to avoid bullets so I'll try this as a comma list.
